src/constants.ts

```diff
@@ -294,68 +380,7 @@ export const DEFAULT_SETTINGS: CopilotSettings = {
   showSuggestedPrompts: true,
   showRelevantNotes: true,
   numPartitions: 1,
-  enabledCommands: {
-    [COMMAND_IDS.FIX_GRAMMAR]: {
-      enabled: true,
-      name: "Fix grammar and spelling of selection",
-    },
-    [COMMAND_IDS.SUMMARIZE]: {
-      enabled: true,
-      name: "Summarize selection",
-    },
-    [COMMAND_IDS.GENERATE_TOC]: {
-      enabled: true,
-      name: "Generate table of contents for selection",
-    },
-    [COMMAND_IDS.GENERATE_GLOSSARY]: {
-      enabled: true,
-      name: "Generate glossary for selection",
-    },
-    [COMMAND_IDS.SIMPLIFY]: {
-      enabled: true,
-      name: "Simplify selection",
-    },
-    [COMMAND_IDS.EMOJIFY]: {
-      enabled: true,
-      name: "Emojify selection",
-    },
-    [COMMAND_IDS.REMOVE_URLS]: {
-      enabled: true,
-      name: "Remove URLs from selection",
-    },
-    [COMMAND_IDS.REWRITE_TWEET]: {
-      enabled: true,
-      name: "Rewrite selection to a tweet",
-    },
-    [COMMAND_IDS.REWRITE_TWEET_THREAD]: {
-      enabled: true,
-      name: "Rewrite selection to a tweet thread",
-    },
-    [COMMAND_IDS.MAKE_SHORTER]: {
-      enabled: true,
-      name: "Make selection shorter",
-    },
-    [COMMAND_IDS.MAKE_LONGER]: {
-      enabled: true,
-      name: "Make selection longer",
-    },
-    [COMMAND_IDS.ELI5]: {
-      enabled: true,
-      name: "Explain selection like I'm 5",
-    },
-    [COMMAND_IDS.PRESS_RELEASE]: {
-      enabled: true,
-      name: "Rewrite selection to a press release",
-    },
-    [COMMAND_IDS.TRANSLATE]: {
-      enabled: true,
-      name: "Translate selection",
-    },
-    [COMMAND_IDS.CHANGE_TONE]: {
-      enabled: true,
-      name: "Change tone of selection",
-    },
-  },
+  enabledCommands: {},
```

Comment:
Change them to be true by default, so there is no need to list them explicitly. I kept the {enable: boolean} structure, so the current user settings will continue to work.